backlog/src/Program.cs

```diff
@@ -29,6 +29,9 @@ public static int Main(string[] args)
 
                 DotNetEnv.Env.Load();  // required for bucket name
 
+                string judgmentsFilePath = Environment.GetEnvironmentVariable("JUDGMENTS_FILE_PATH");
```

Comment:
How many of these are we going to have? Do we need a more sustainable model?

Reply:
as in how many env vars or how many potentially different judgmentsFilePath values from the different tribunals?

Env vars: there may be some other things that are hardcoded and we want to generalise but hopefully not many more

different judgmentsFilePath from tribunals: potentially lots but this means we are agnostic to that

Reply:
will continue discussion on slack